src/ports/postgres/modules/assoc_rules/test/assoc_rules.sql_in

```diff
@@ -137,29 +144,63 @@ begin
           assoc_array_eq(t1.post, t2.post) AND
           abs(t1.support - t2.support) < 1E-10 AND
           abs(t1.confidence - t2.confidence) < 1E-10;
+    IF (result2 = 'FAIL') THEN
```

Comment:
Do we have tests that check for the support and confidence values ? If not, it might be a good time to add them to prevent any future regressions

Reply:
I am not sure what you mean. This is literally the test that check for the support and confidence for each rule. It counts the rules that match the fixed values in the expected results table `test2_exp_result` and writes PASS or FAIL to a table if this count is 7. Unless the rules as well as the support and confidence levels match, the result will be a failure. I guess it is a bit unconventional among madlib tests and we might think about changing it in the future.

Reply:
ohh I see I just saw the exception and thought this was testing some error condition.